### PR TITLE
scylla_node: improve the sample return val of node.dump_sstable()

### PR DESCRIPTION
**Pull request by tchaikov**
so we can have better understanding of the format when using this function.

### CODE REVIEW

**Comment by tchaikov** [comment]
inspired by https://github.com/scylladb/scylla-dtest/pull/3487 , where we inspect the ttl of a cell.

**Review by denesb — APPROVED** [review]
Maybe add a link to https://opensource.docs.scylladb.com/stable/operating-scylla/admin-tools/scylla-sstable.html#dump-data, where the format is documented.

**Comment by tchaikov** [comment]
v2:

- reference the link to the document of scylla-sstable

**Comment by tchaikov** [comment]
@denesb thanks! revised to add the link.

File: ccmlib/scylla_node.py [hunk 1]
```diff
@@ -1455,6 +1455,10 @@ def dump_sstables(self,
         :return: return all the partitions collected in the specified sstables
         :raises: subprocess.CalledProcessError if scylla-sstable returns a non-zero exit code.
 
+        only the $PARTITION is returned from this method, please see
```
Comment (denesb) [inline ccmlib/scylla_node.py:1458]: Isn't it the `$SSTABLE` symbol in fact that we return here?
```
$SSTABLE := [$PARTITION, ...]
```
We return multiple partitions.

Comment (tchaikov) [reply ccmlib/scylla_node.py]: ahh, indeed!

**Comment by tchaikov** [comment]
v3:

- s/PARTITION/SSTABLE/ in the comment

**Comment by tchaikov** [comment]
@scylladb/qa-maintainers hello maintainers, could you help merge this change?

**Pull request merged**